test/integration/discord.test.js

```diff
@@ -0,0 +1,103 @@
+const chai = require("chai");
+const { expect } = chai;
```

Comment:
did we have sinon preinstalled or you installed it?

Reply:
It was preinstalled